pkg/cmd/tests/tests_run.go

```diff
@@ -309,6 +309,63 @@ func (o *RunOptions) run(ctx context.Context, streams genericclioptions.IOStream
 
 	gomega.RegisterFailHandler(ginkgo.Fail)
 
+	// Temporary workaround for https://github.com/scylladb/scylla-operator/issues/1176.
+	// Run a check after suite has completed to verify whether all specs were run by a single process.
+	// TODO: remove once #1176 has been fixed.
+	ginkgo.ReportAfterSuite("Verify report", func(r ginkgo.Report) {
+		expectedSpecs := []string{
```

Comment:
how does this work for different suites or for feature flags?

Reply:
Every suite, regardless of the feature flags, builds the same spec tree - they just have a different status in the final report. The only difference is the status of the nodes - different subsets are ran/skipped. Only in case of an invalid parallel run the specs are missing from the final report, as multiple processes get the same subset of specs.

Reply:
didn't the #1245 had the same number of overall test just 2 more were skipped instead of being run?

Reply:
There was a higher number of skipped specs than usual but that's coincidental - the duplicated tests that were selected instead of the missed ones happened to be the ones being skipped.